packages/shared/src/authz/fragments/projects.spec.ts

```diff
@@ -16,11 +17,12 @@ import {
   WorkspaceSsoSessionNoAccessError
 } from '../domain/authErrors.js'
 import { OverridesOf } from '../../tests/helpers/types.js'
+import { getProjectFake } from '../../tests/fakes.js'
 
 describe('ensureMinimumProjectRoleFragment', () => {
   const buildSUT = (overrides?: OverridesOf<typeof ensureMinimumProjectRoleFragment>) =>
     ensureMinimumProjectRoleFragment({
-      getProject: async () => ({
+      getProject: getProjectFake({
```

Comment:
we should use getProjectFake/getComentFake/GetXXXFake everywhere to avoid having to update a bunch of places in code just because we've added a new prop to the underlying resource types